src/app/zap-templates/zcl/data-model/chip/rvc-run-cluster.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ChangeToModeResponseStatus" type="ENUM8">
+    <cluster code="0x0054"/>
+    <item value="0x40" name="Battery Low"/>
```

Comment:
Again, all of the spaces should go away in the names, but followup is ok for that.

Reply:
accetped.